charts/rucio-server/values.yaml

```diff
@@ -184,6 +185,7 @@ wsgi:
   daemonProcesses: "4"
   daemonThreads: "4"
 
+## values used to configure apache
 httpd_config:
   mpm_mode: "event"
```

Comment:
should this be in the flux config or do we want to use the event mode by default?

Reply:
mpm_event is the best mode that apache has, so can/should be default. there's no upsides to using mpm_worker or mpm_prefork

Reply:
you are right, I didn't realize that removing the duplicate key (httpd_config) in the yaml below will result in a change of behavior.